packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js

```diff
@@ -484,23 +484,22 @@ export default class WalletSendForm extends Component<Props> {
             options={sendAmountOptions}
             {...form.$('selectedAmount').bind()}
             onChange={value => {
-              this.form.$('selectedAmount').value = value
-              if(value === CUSTOM_AMOUNT && this.props.shouldSendAll) {
-                this.props.toggleSendAll();
+              if(value === CUSTOM_AMOUNT) {
+                this.props.updateSendAllStatus(false);
+                this.form.$('amount').clear()
               } else if (value !== CUSTOM_AMOUNT) {
-                this.props.toggleSendAll()
-              }
-              if (this.props.shouldSendAll) {
-                // if we toggle shouldSendAll from true -> false
-                // we need to re-enable the field
-                // and set it to whatever value was used for the sendAll value
-                this.props.updateAmount(new BigNumber(
-                  formattedAmountToNaturalUnits(
-                    this.form.$('amount').value,
-                    this.getNumDecimals(),
-                  )
-                ));
+                this.props.updateSendAllStatus(true)
+                // Will only update the amount to the max when we switch from custom amount to send all
```

Comment:
No. The original behavior is to update the amount to the (auto-calculated) max when switching *from send all to custom amount*. Please resume the original behavior.